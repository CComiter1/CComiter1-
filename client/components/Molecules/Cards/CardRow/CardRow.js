import React from 'react'
import styled from 'styled-components'

import {
  CardBody,
  CardBodyBold,
} from '../../../Atoms'

const Row = styled.div`
  display: flex;
  justify-content: space-between;
  width: 100%;
  padding: 20px 0;  
`

export default props => (
  <Row>
    <CardBodyBold>{props.label}:</CardBodyBold>
    <CardBody>{props.children}</CardBody>
  </Row>
)
