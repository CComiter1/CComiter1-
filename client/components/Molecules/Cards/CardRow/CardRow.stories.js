import React from 'react'
import { storiesOf } from '@storybook/react'

import CardRow from './CardRow'

storiesOf('Molecules/Cards', module)
  .add('Card Row', () => <CardRow label="Quantity">10</CardRow>)
