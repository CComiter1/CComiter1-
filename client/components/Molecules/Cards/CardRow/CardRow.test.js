import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardRow from './CardRow'

describe('CardRow', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <CardRow label="label text">children text</CardRow>
    ).toJSON()
    expect(component).toMatchSnapshot()
  })

  it('loads two instances of CardBody', () => {
    expect(shallow(<CardRow label="label">text</CardRow>).children().length).toBe(2)
  })
})
