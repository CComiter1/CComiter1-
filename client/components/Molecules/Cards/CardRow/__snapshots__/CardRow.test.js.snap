// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardRow renders correctly 1`] = `
.c2 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
}

.c1 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
  font-weight: bold;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  padding: 20px 20px;
}

<div
  className="c0"
>
  <p
    className="c1"
  >
    label text
    :
  </p>
  <p
    className="c2"
  >
    children text
  </p>
</div>
`;
